### PR TITLE
Adds a new button for admins, the SMITE BUTTON

### PR DESCRIPTION
**Pull request by lzimann**
:cl: Lzimann
:add: The gods can now punish you if you annoy them too much with prayers!
/:cl:

The next step in admin abuse, a more "IC" punishment for people that are annoying with prayers(where is the fun in banning them?!)!

Current available punishments:
Lightning bolt: Does 75 burn damage.
Brain damage: Adds 75 brain damage to the prayer
Gib: Gibs(without dropping a brain)

### CODE REVIEW

**Comment by MimicFaux** [comment]
Merge immediately

**Comment by PKPenguin321** [comment]
Just what I needed

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1700,12 +1700,42 @@
...
+		if(!check_rights(R_ADMIN|R_FUN))
+			return
+
+		var/mob/living/carbon/human/H = locate(href_list["adminsmite"]) in mob_list
+		if(!H || !istype(H))
+			return
+
+		var/list/punishment_list = list(ADMIN_PUNISHMENT_LIGHTNING, ADMIN_PUNISHMENT_BRAINDAMAGE, ADMIN_PUNISHMENT_GIB)
+
+		var/punishment = input("Choose a punishment", "DIVINE SMITING") as null|anything in punishment_list
```
Comment (optimumtact) [inline code/modules/admin/topic.dm]: check target and admin exist and are not qdeleted after this input call

**Comment by Haevacht** [comment]
Should have a few varied methods to suit whatever meme bullshit god the chaplain worships at the time.

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (Jalleo) [inline code/modules/admin/topic.dm]: Should also add in a finger of god smiting (monty python style)

**Comment by lzimann** [comment]
They can be easily added in the future if anyone has a good idea.

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm giving every chaplain who prays to X when their chosen god is Y brain damage.

**Pull request merged**